cfy_manager/components/validations.py

```diff
@@ -417,6 +417,25 @@ def _validate_ssl_and_external_certificates_match():
                               "`ssl_enabled` to true.")
 
 
+def _is_cert_self_signed(cert_file):
+    response = os.system('openssl verify -CAfile {0} {0}'.format(cert_file))
+    if response == 0:  # success
```

Comment:
Is response from os.system just the return code? So it is.
Is there a reason for using this rather than our built-in run utility methods? You can either pass them ignore_failures=True (which will mean that you get response.returncode as the exit status), or try then catch the error.

I suggest this because that way whatever we do with regards to logging commands we run, etc, only has to happen in one place.

Also, a simpler way of stating the return if you have just the returncode (e.g. response.returncode) is:
return response.returncode == 0

(or, in the way it's currently written: return response == 0)

Reply:
And yes, I should've suggested using the run functions from utils before, sorry!

Reply:
No worries, I'll fix it now, thanks!

Reply:
Done